lib/core/pipes/localized-date.pipe.ts

```diff
@@ -59,7 +59,7 @@ export class LocalizedDatePipe implements PipeTransform, OnDestroy {
         const actualFormat = format || this.defaultFormat;
         const actualLocale = locale || this.defaultLocale;
         const datePipe: DatePipe = new DatePipe(actualLocale);
-        return datePipe.transform(value, actualFormat);
+        return datePipe.transform(value, actualFormat, null, actualLocale);
```

Comment:
I believe we don't need that last actualLocale property as we create the pipe with it in the previous line.

Reply:
Updated.